src/Common/Document.php

```diff
@@ -36,4 +36,9 @@ public function getSource(): array
     {
         return $this->source;
     }
+
+    public function getEmsLink(): EMSLink
+    {
+        return EMSLink::fromText("ems://object:$this->contentType:$this->ouuid");
```

Comment:
Do not like this, may sprintf?

Reply:
Or a new static contructor on EMSLink -> EMSLink::fromContentTypeOuuid($this->contentType, $this->ouuid)